addon/components/frost-modal-dialog.js

```diff
@@ -1,14 +1,32 @@
+/* global Ps */
 import Ember from 'ember'
-import layout from '../templates/components/frost-modal-dialog'
 import computed from 'ember-computed-decorators'
+import PropTypeMixin, {PropTypes} from 'ember-prop-types'
```

Comment:
I updated ember-prop-types to add itself to the component in an initializer. If you want you can change this to just import the PropTypes:

`import {PropTypes} from 'ember-prop-types'`